drgn_tools/table.py

```diff
@@ -42,3 +43,37 @@ def print_table(
 
     if outfile:
         out.close()
+
+
+def print_dictionary(
+    dictionary: Dict[str, Any],
+    outfile: Optional[str] = None,
+    report: bool = False,
+) -> None:
+    """
+    Align and print the data
+
+    :param dictionary: dictionary to print
+    :param outfile: A file to write the output to.
+    :param report: Open the file in append mode.
+    :returns: None
+    """
+    lcol_length = 10
+    for title in dictionary:
+        lcol_length = max(len(title), lcol_length)
+
+    out = sys.stdout
+    if outfile and report:
+        out = open(outfile, "a")
+        print("\n", file=out)
+    elif outfile:
+        out = open(outfile, "w")
+
+    for title in dictionary:
+        print(
+            f"{title.ljust(lcol_length)}: {dictionary[title]:<}",
```

Comment:
`{dictionary[title]:<}`
nit: technically, left alignment is the default, so there's no need to specify it